.github/workflows/mutation-testing.yml

```
@@ -0,0 +1,80 @@
+
+env:
+  MUTATION_TOOLCHAIN: nightly-2025-07-14
+
+jobs:
+  cargo-mutants:
+    name: cargo-mutants (${{ matrix.label }})
+    runs-on: ubuntu-22.04
+    timeout-minutes: 360
+    env:
+      MUTATION_SHARD: ${{ matrix.shard }}
+      MUTATION_TOOLCHAIN: ${{ env.MUTATION_TOOLCHAIN }}
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)